### PR TITLE
[1LP][RFR] Fixing 5.10 failures

### PR DESCRIPTION
**Pull request by Yadnyawalkya**
{{ pytest: cfme/tests/v2v/test_v2v_migrations.py --use-provider rhv43 --use-provider vsphere65-nested --provider-limit 2 -vvvv -k 'test_single_datastore_single_vm_migration or test_single_network_single_vm_migration or test_dual_nics_migration' }}

Fixing,
- Migration with local datastore failures
- Network map migration failures
- Dual NIC failures
- Update of migration plan collection
- Duplicate tests

### CODE REVIEW

File: cfme/v2v/infrastructure_mapping.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ class NetworkComponent(InfraMappingComponent):
     datastores = attr.ib(validator=attr.validators.instance_of(list))
     networks = attr.ib(validator=attr.validators.instance_of(list))
 
-    def fill_dict(self):
+    def fill_dict(self, target_selection=True):
```
Comment (sshveta) [inline cfme/v2v/infrastructure_mapping.py]: What is this for ? Target_selection ?
We are always selecting target .Aren't we ?

Comment (Yadnyawalkya) [reply cfme/v2v/infrastructure_mapping.py]: by default we selects target but when we edit map we can not edit this selection

Screenshot from 2019-07-16 13-08-55

**Comment by mshriver** [comment]
5.11 is failing to add RHV in PRT, but that's not related to any changes here.

**Comment by Yadnyawalkya** [comment]
Thanks @mshriver.

**Pull request merged**